docs/domain-model.md

```diff
@@ -0,0 +1,123 @@
+# Domain Model
+---
+
+## Definitions:
+
+### EDC connector
+
+<p style='text-align: justify;'> EDC is an interoperable, cross-organization framework for data sharing between 2 or more Organizations in a trusted and
```

Comment:
The domain model file should not explain what the EDC is. This is duplicated information, and if not, should be placed elsewhere.